android/build.gradle

```diff
@@ -57,6 +57,8 @@ dependencies {
     implementation 'com.github.bumptech.glide:annotations:4.11.0'
     implementation 'com.github.bumptech.glide:glide:4.11.0'
 
+    implementation 'io.reactivex.rxjava2:rxjava:2.1.13'
```

Comment:
Based on `zoom-sdk-android-5.16.2.16555:mobilertc-android-studio/mobilertc/build.gradle` it looks like we should rather include `dependencies.add("default","io.reactivex.rxjava2:rxandroid:2.1.1")`